src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -62,17 +62,17 @@ def test_csum_error_logging(self):
         host_devices = get_dmg_smd_info(dmg.storage_query_list_devices, 'devices')
         for host, devices in host_devices.items():
             for device in devices:
-                for entry in ('uuid', 'tgt_ids', 'role_bits', 'roles'):
+                for entry in ('uuid', 'tgt_ids', 'role_bits'):
                     if entry not in device:
                         self.fail(
                             'Missing {} info from dmg storage query list devices'.format(entry))
                 self.log.info(
-                    'Host %s device: uuid=%s, targets=%s, role=%s, role_bits=%s',
-                    host, device['uuid'], device['tgt_ids'], device['roles'], device['role_bits'])
+                    'Host %s device: uuid=%s, targets=%s, role_bits=%s',
+                    host, device['uuid'], device['tgt_ids'], device['role_bits'])
```

Comment:
Similarly this change should be reverted:
```suggestion
                    'Host %s device: uuid=%s, targets=%s, role=%s, role_bits=%s',
                    host, device['uuid'], device['tgt_ids'], device['roles'], device['role_bits'])
```

Reply:
I am going with Tom's change to make the test more flexible and generic.